.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: go
 
 go:
-  - 1.14
+  - 1.15
```

Comment:
Since 1.16 was released already, should these bumps go directly to 1.16?

Reply:
That sounds good to me!